src/NuGet.CommandLine.StagingExtensions/StageCommand.cs

```diff
@@ -146,6 +179,144 @@ private bool TryParseArguments(out SubCommand subCommand, out string parameter)
             }
         }
 
+        private async Task GetStageDetails(StageManagementResource stageManagementResource, string stageId)

```

Comment:
it outputs stage details to the console.

---
In reply to: [78215929](https://github.com/NuGet/NuGet.Staging/pull/73#discussion_r78215929) [](ancestors = 78215929)